.github/workflows/ubuntu.yml

```diff
@@ -156,17 +154,17 @@ jobs:
       - name: archive-files
         # We only run tests on the version compiled with gcc, so we
         # only need to archive that one.
-        if: ${{ matrix.compiler-version }} == gcc
+        if: ${{ matrix.compiler-version.cc == 'gcc' }}
         run: |
           cd ../
-          tar cfz ${{ matrix.compiler-version }}.tar.gz downward lib
+          tar cfz ${{ matrix.compiler-version.cc }}.tar.gz downward lib
 
       - name: upload-files
-        if: ${{ matrix.compiler-version }} == gcc
+        if: ${{ matrix.compiler-version.cc == 'gcc' }}
         uses: actions/upload-artifact@v1
         with:
           name: compiled-planner-${{ matrix.ubuntu-version }}
-          path: /home/runner/work/downward/${{ matrix.compiler-version }}.tar.gz
+          path: /home/runner/work/downward/${{ matrix.compiler-version.cc }}.tar.gz
 
 
   test:
```

Comment:
Why do we need to specify and install the compiler for the tests?

Reply:
I think because some tox tests (search?) also check if the code is built. If CMAKE doesn't find compilers, it complains. Maybe we would not need to specify a spefific version, but it also doesn't hurt to always use the same setup.

Reply:
I see. Can you add this info as a comment so we don't wonder about this in the future?